diff --git a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/LoanProductsRequestFactory.java b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/LoanProductsRequestFactory.java
index d91464abf..29db7cfc9 100644
--- a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/LoanProductsRequestFactory.java
+++ b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/LoanProductsRequestFactory.java
@@ -96,7 +96,7 @@ public class LoanProductsRequestFactory {
     public static final String DESCRIPTION_INTEREST_DECLINING_BALANCE_DAILY_RECALCULATION_COMPOUNDING_MONTHLY = "LP1-1MONTH with 12% DECLINING BALANCE interest, interest period: Daily, Interest recalculation-Monthly, Compounding:Interest";
     public static final String DESCRIPTION_INTEREST_DECLINING_BALANCE_DAILY_RECALCULATION_COMPOUNDING_NONE = "LP1 with 12% DECLINING BALANCE interest, interest period: Daily, Interest recalculation-Daily, Compounding:none";
     public static final Long FUND_ID = FundId.LENDER_A.value;
-    public static final String CURRENCY_CODE = "EUR";
+    public static final String CURRENCY_CODE = "MNT";
     public static final String CURRENCY_CODE_USD = "USD";
     public static final Integer INTEREST_RATE_FREQUENCY_TYPE_MONTH = InterestRateFrequencyType.MONTH.value;
     public static final Integer INTEREST_RATE_FREQUENCY_TYPE_YEAR = InterestRateFrequencyType.YEAR.value;
diff --git a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/SavingsProductRequestFactory.java b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/SavingsProductRequestFactory.java
index 968448dff..9419bbfe7 100644
--- a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/SavingsProductRequestFactory.java
+++ b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/SavingsProductRequestFactory.java
@@ -30,7 +30,7 @@ public final class SavingsProductRequestFactory {
     public static final String DEFAULT_SAVINGS_PRODUCT_NAME = "CEUR";
     public static final String DEFAULT_SAVINGS_PRODUCT_SHORT_NAME = "CEU";
     public static final String DEFAULT_SAVINGS_PRODUCT_DESCRIPTION = "";
-    public static final String DEFAULT_SAVINGS_PRODUCT_CURRENCY_CODE = "EUR";
+    public static final String DEFAULT_SAVINGS_PRODUCT_CURRENCY_CODE = "MNT";
     public static final Integer DEFAULT_SAVINGS_PRODUCT_DIGITS_AFTER_DECIMAL = 2;
     public static final Integer DEFAULT_SAVINGS_PRODUCT_IN_MULTIPLES_OF = 0;
     public static final Double DEFAULT_SAVINGS_PRODUCT_NOMINAL_ANNUAL_INTEREST_RATE = 0.0;
diff --git a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/WorkingCapitalChargeRequestFactory.java b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/WorkingCapitalChargeRequestFactory.java
index a35df0968..4885d26a9 100644
--- a/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/WorkingCapitalChargeRequestFactory.java
+++ b/fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/factory/WorkingCapitalChargeRequestFactory.java
@@ -28,7 +28,7 @@ import org.springframework.stereotype.Component;
 @Component
 public class WorkingCapitalChargeRequestFactory {
 
-    public static final String DEFAULT_CURRENCY_CODE = "EUR";
+    public static final String DEFAULT_CURRENCY_CODE = "MNT";
     public static final String DEFAULT_LOCALE = "en";
     public static final Double DEFAULT_AMOUNT = 20.0D;
     public static final String DEFAULT_NAME_PREFIX = "WCL_CHARGE_";
diff --git a/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/ChargeGlobalInitializerStep.java b/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/ChargeGlobalInitializerStep.java
index ed2382353..5fa727344 100644
--- a/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/ChargeGlobalInitializerStep.java
+++ b/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/ChargeGlobalInitializerStep.java
@@ -35,7 +35,6 @@ import org.apache.fineract.test.data.ChargePaymentMode;
 import org.apache.fineract.test.data.ChargeProductAppliesTo;
 import org.apache.fineract.test.data.ChargeProductType;
 import org.apache.fineract.test.data.ChargeTimeType;
-import org.apache.fineract.test.data.CurrencyOptions;
 import org.apache.fineract.test.helper.ErrorMessageHelper;
 import org.apache.fineract.test.helper.ParallelExecutionHelper;
 import org.apache.fineract.test.support.TestContext;
@@ -47,7 +46,7 @@ import org.springframework.stereotype.Component;
 @Component
 public class ChargeGlobalInitializerStep implements FineractGlobalInitializerStep {
 
-    public static final String CURRENCY_CODE = CurrencyOptions.EUR.value;
+    public static final String CURRENCY_CODE = "MNT";
     public static final String MONTH_DAY_FORMAT = "dd MMM";
     public static final String LOCALE_EN = "en";
     public static final Integer CHARGE_PAYMENT_MODE = ChargePaymentMode.REGULAR.value;
diff --git a/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/CurrencyGlobalInitializerStep.java b/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/CurrencyGlobalInitializerStep.java
index 4b8f73a52..6a5f748b4 100644
--- a/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/CurrencyGlobalInitializerStep.java
+++ b/fineract-e2e-tests-runner/src/test/java/org/apache/fineract/test/initializer/global/CurrencyGlobalInitializerStep.java
@@ -37,7 +37,7 @@ import org.springframework.stereotype.Component;
 @Order(Ordered.HIGHEST_PRECEDENCE)
 public class CurrencyGlobalInitializerStep implements FineractGlobalInitializerStep {
 
-    public static final List<String> CURRENCIES = Arrays.asList("EUR", "USD");
+    public static final List<String> CURRENCIES = Arrays.asList("MNT", "USD");
 
     private final FineractFeignClient fineractClient;
 
